doc/developer-guide/developer-guide.rst

```diff
@@ -14,11 +14,11 @@ Dependencies
 
 These are the dependencies for the latest version of ts_dream:
 
-* ts_xml - v9.0.0
+* ts_xml - v9.1
 * `SAL <https://ts-sal.lsst.io>`_ - v5.1.1
-* ts_salobj - v6.4.1
-* ts_idl - v3.1.3
-* ts_tcpip - v0.3.0
+* ts_salobj - v6.5
+* ts_idl - v3.2
+* ts_tcpip - v0.3
```

Comment:
what about `ts-tcpip`?

Reply:
Do you mean ts_dream_common? Yeah, I'll add that.